protocol-units/da/m1/light-node/src/v1/sequencer.rs

```diff
@@ -200,14 +200,13 @@ impl LightNodeService for LightNodeV1 {
 			.collect::<Result<Vec<BlobResponse>, tonic::Status>>()?;
 
 		// make transactions from the blobs
-		let transactions: Vec<Transaction> = blobs_for_submission
-			.into_iter()
-			.map(|blob| {
-				let transaction = Transaction::from(blob.data);
-				transaction
-			})
-			.collect();
-
+		let mut transactions = Vec::new();
+		for blob in blobs_for_submission {
+			let transaction : Transaction = serde_json::from_slice(&blob.data)
+				.map_err(|e| tonic::Status::internal(e.to_string()))?;
```

Comment:
This added layer of JSON formatting in what used to be a blob bothers me a little, too.
Can we work the sequence number into the gRPC message instead, and keep the transaction data as a separate `bytes` field?

Reply:
That's what I was asking about the other day. We could do that. Given the hour, it may be better to do it in a separate PR after CI issues are sorted here. That way we can push this to update the devnet.

Reply:
Filed as #182.